include/SFML/Audio/SoundStream.hpp

```diff
@@ -251,6 +251,24 @@ class SFML_AUDIO_API SoundStream : public SoundSource
     ////////////////////////////////////////////////////////////
     virtual Int64 onLoop();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the processing interval
+    ///
+    /// The processing interval controls the period
+    /// between calls to the fillAndPushBuffer function. You may
```

Comment:
You should rather talk about `getData`, `fillAndPushBuffer` means nothing for the user.